README.md

```diff
@@ -82,4 +85,4 @@ Lima follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/
   <img src="https://www.cncf.io/wp-content/uploads/2022/07/cncf-color-bg.svg" width=300 />
 </picture>
 
-The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For a list of TLF trademarks, see [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/).
+The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For a list of TLF trademarks, see [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/].
```

Comment:
This is a bogus change by Copilot Workspace; please double-check the changes, sometimes it just breaks something that was already correct.